dev-resources/bootstrapping/cli/bootstrap_with_comments.cfg

```diff
@@ -0,0 +1,5 @@
+# commented out line
+puppetlabs.trapperkeeper.examples.bootstrapping.test-services/hello-world-service # comment
+; another commented out line
+ ;puppetlabs.trapperkeeper.examples.bootstrapping.test-services/foo-test-service
```

Comment:
This has a space before the comment - not sure that it matters? but figured I would point it out cuz it seems a little inconsistent.